common/scripted_effects/wc_artifacts_scripted_effects.txt

```diff
@@ -403,8 +433,770 @@ spawn_artifact_ring_5_intrigue_effect = {
 spawn_artifact_ring_5_learning_effect = {
 	add_artifact = ring_5_learning
 	new_artifact = {
-		set_prefix_and_suffix_temp_flags_to_artifact_effect = yes
+		set_prefix_and_suffix_temp_flags_to_jewelry_artifact_effect = yes
 		set_name="ring_5_learning"
 		finish_artifact_creation_effect = yes
 	}
+}
+
+# Two-handed Swords
+
+spawn_artifact_sword_twohanded_1_effect = {
+	add_artifact = sword_twohanded_1
+	new_artifact = {
+		set_prefix_and_suffix_flags_to_weapon_artifact_effect = yes
+		set_name="sword_twohanded_1"
+		finish_artifact_creation_effect = yes
+	}
+}
+
+spawn_artifact_sword_twohanded_2_effect = {
+	add_artifact = sword_twohanded_2
+	new_artifact = {
+		set_prefix_and_suffix_flags_to_weapon_artifact_effect = yes
+		set_name="sword_twohanded_2"
+		finish_artifact_creation_effect = yes
+		if = {
+			limit = { has_artifact_flag = artifact_weapon_element_fire_flag }
+			set_picture = "GFX_sword_twohanded_2_fire"
+		}
+		else_if = {
+			limit = { has_artifact_flag = artifact_weapon_element_frost_flag }
+			set_picture = "GFX_sword_twohanded_2_cold"
+		}
+		else_if = {
+			limit = { has_artifact_flag = artifact_weapon_element_lightning_flag }
+			set_picture = "GFX_sword_twohanded_2_lightning"
+		}
+		else_if = {
+			limit = { has_artifact_flag = artifact_weapon_element_poison_flag }
+			set_picture = "GFX_sword_twohanded_2_poison"
+		}
+		else = {
+			limit = { has_artifact_flag = artifact_weapon_element_magic_flag }
+			set_picture = "GFX_sword_twohanded_2_magic"
+		}
+	}
+}
```

Comment:
`artifact_weapon_element_fire_flag` etc. not needed after the picture is set, right? So suggest move `finish_artifact_creation_effect` after the picture setting and clear these flags there.

Reply:
> artifact_weapon_element_fire_flag etc. not needed after the picture is set, right?

Will be needed in distant future for artifacts upgrading, so they don't change element after upgrade.